### ISSUE TITLE
CalTRACK Issue: Difference in Differences (DiD) methods using control groups for better gross / net savings 

### ISSUE THREAD

**Issue by AJLutz** [energy-market-methods/caltrack#119]
**Issue:** Difference in Differences (DiD) methods using control groups can yield more accurate and discrete gross and net savings estimates than normalized pre and post intervention comparisons for participant groups only. 

Prerequisites: This issue is in the CalTrack working group. Issues previously opened are only tangential to the formulation of control groups [exogenous effects (#1050, accuracy (#73), zip code mapping (#65, # 26) and non- routine events (#84)]. 

Description: The proposed update would require consideration of appropriate control groups and their use when valid control groups can be developed. 

Using before and after consumption normalized for weather (and possibly other variables) provide estimates of total savings for the participant group. This includes gross impacts, net impacts and ‘exogenous’ impacts due to the economy, societal messaging, customer behavior and other events.  These exogenous effects can be captured using a control group with DiD mthods, which can help isolate those net (free ridership) impacts and provide more reliable net to gross (NTG) impacts for specific interventions. NTG data gathering can then concentrate on more definitive and discrete events as to whether the control group member implemented a specific intervention outside of the program, replacing more ambiguous and costly analysis methods. Note that if the program intervention was implemented by a control group member during the pre or post period, net impacts could be slightly underestimated as the potential implementing non-participant pool is smaller (exogenous effects would not be affected).
Methods for matching participants can be standardized based on available information. 

Matching criteria can include zip codes, groups of zip codes, county, climate zones, population density (large urban/small urban/suburban, exurban, rural designation), household size, income levels, building square footage, building type (SF/MF/DMo), ownership (own/rent), condo flag, change in service ID during pre or post analysis period, and daily/monthly energy use profiles (as a proxy for some of the aforementioned criteria). Control group members would be dropped upon change in Service ID, major other EE/RE/electrification measures, or participation in other correlated EE programs during the pre or post period.

Proposed Test Methodology
1)	Develop minimum population of recent program participants (2016/2017, residential, similar characteristics from description above). Smaller sample sizes (minimum 100) are allowed for test phase. For testing purposes, size can be small and restricted to one zip code and building type, for instance. 
2)	Find matching sample from non-participating customers. Verify no change in ownership or major energy using equipment (including adoption of incented technologies).
3)	Use the same weather adjustments for control group as applied to participant group (or other appropriate identical weather adjustments for both groups). 
4)	Test that regression models for the participant group function for the control group (+/-10%; include r squared and CVRSME at a minimum).
5)	Check proportion of savings from participants and control groups.
6)	Make adjustments based on findings until successful for small test populations. Expend to large populations (segmented as needed). Test regression models.

Acceptance Criteria
 If regression model results are similar for the larger set of participants, accept the use of control groups to account for both net impacts (free ridership) and exogenous effects.  Gross impacts for the participants can be adjusted upwards or downwards based on the results from the control group.  (Findings should be presented with program reporting to the CPUC for more accurate and refined net to gross analysis).

**Comment by mcgeeyoung** [comment]
@AJLutz Thanks for this issue. It's actually out of scope of CalTRACK, but perfect for GRID. Would you mind reposting it in that working group? https://github.com/energy-market-methods/GRID/issues

Thanks,
McGee

**Comment by mcgeeyoung** [comment]
@jkoliner Once Al moves this to GRID, where it belongs, we should have some way of closing and marking as out of scope. Maybe a tag of sorts. @hshaban Does that make sense to you?

**Comment by mcgeeyoung** [comment]
One more point @AJLutz when you do repost this in GRID, it would be helpful to be a lot more specific. Take a look at the existing CalTRACK methods for a reference. We'd need to document how exactly the comparison group would be selected. What exactly would be the acceptance criteria, etc.

**Comment by AJLutz** [comment]
McGee –

I can see how the net impacts may fall under GRID.

However, with the larger samples afforded under CalTrack, and the fact that we are up and going there, I still think it is relevant there. Maybe we can list in both places?

Control groups would seem to fall under CalTrack as they are a tool to help us get to the right values. Some jurisdictions claim a NTGR of 1.00, so we’d be seeing exogenous effects only. I see a direct comparison to the pharma industries that use control groups most often (with millions of lives and billions in revenues on the line). Why not in CalTrack?

Maybe it will not raise to the top issues to be addressed in the first round for CalTrack. We discussed tackling three issues at a time.

As a pre draft issue, I am not looking to develop this further for GRID or CalTrack. Can further detail wait until the groups decide when this is to be tackled?

Al

Al Lutz, PE, CEM, LEED AP
AJL Resources, LLC
510-504-4813
www.al-lutz.com
www.linkedin.com/in/al-lutz-ajlenergy<http://www.linkedin.com/in/al-lutz-ajlenergy>

PS - Sorry I have a conflict and may not be able to make the GRID meeting (or not until the end). Please let me know if there is relevant discussion on this.

**Comment by hshaban** [comment]
Closed because it's been resubmitted to GRID Working group
